test/commands/init_test.js

```diff
@@ -10,258 +10,190 @@
-          'polymer-init-app-drawer-template:app'
-        ));
-
-        assert.isTrue(inquirerStub.prompt.calledWithMatch(match(function (value) {
-          assert.sameDeepMembers(
-            value[0].choices,
-            [
-              {
-                name: 'element: A blank element template',
-                value: 'polymer-init-element:app',
-                short: 'element',
-              },
-              {
-                name: 'application: A blank application template',
-                value: 'polymer-init-application:app',
-                short: 'application',
-              },
-              {
-                name: 'shop: The "Shop" Progressive Web App demo',
-                value: 'polymer-init-shop:app',
-                short: 'shop',
-              },
-              {
-                name: 'app-drawer-template: A starter application template, ' +
-                'with navigation and "PRPL pattern" loading',
-                value: 'polymer-init-app-drawer-template:app',
-                short: 'app-drawer-template',
-              },
-            ]
-          );
-          return true;
-        })));
-      });
-  });
-
-  test('prompts with a list of all registered generators', () => {
-    mockPlatform('linux');
-    yeomanEnvStub.prototype.getGeneratorsMeta.returns({
-      'polymer-init-foo:app': {},
-      'polymer-init-bar:app': {},
-    });
-    stubInquirer('polymer-init-foo:app');
-
-    return new InitCommand().run({}, {})
-      .then(() => {
-        assert.isTrue(inquirerStub.prompt.calledWith([
           {
-            type: 'list',
-            name: 'generatorName',
-            message: 'Which starter template would you like to use?',
-            choices: [
-              {
-                "name": "foo: no description",
-                "value": "polymer-init-foo:app",
-                "short": "foo",
-              },
-              {
-                "name": "bar: no description",
-                "value": "polymer-init-bar:app",
-                "short": "bar",
-              },
-            ],
+            name: 'custom-template: \u001b[2mno description\u001b[22m',
+            value: 'polymer-init-custom-template:app',
+            short: 'custom-template',
           },
-        ]));
-      });
+        ],
+      },
+    ]));
   });
 
-  test('prompts with a rawlist if being used in MinGW shell', () => {
-    mockPlatform('windows');
-    execStub.execSync.returns('mingw');
-
-    yeomanEnvStub.prototype.getGeneratorsMeta.returns({
-      'polymer-init-foo:app': {},
-      'polymer-init-bar:app': {},
+  if (isPlatformWin) {
+
+    test('prompts with a rawlist if being used in MinGW shell', () => {
+      let promptStub = sandbox.stub(inquirer, 'prompt').returns(new Promise(function() {}));
+      sandbox.stub(childProcess, 'execSync').withExactArgs('uname -s').returns('mingw');
+      cli.run();
+
+      assert.isTrue(promptStub.calledWith([
+        {
+          "type":"rawlist",
```

Comment:
The only thing we care about here is the type, no? Can we make the test smaller so it's less brittle to changes in the set of generators and their descriptions, etc?